lib/ansible/modules/cloud/kubevirt/kubevirt_raw.py

```diff
@@ -20,99 +20,37 @@
     - Pass the object definition from a source file or inline.
     - Authenticate using either a config file, certificates, password or token.
 
-version_added: "2.5"
+version_added: "2.8"
 
 author: KubeVirt Team (@kubevirt)
 
+extends_documentation_fragment:
+  - k8s_state_options
+  - k8s_name_options
+  - k8s_resource_options
+  - k8s_auth_options
+
 options:
+  merge_type:
+    description:
+    - Whether to override the default patch merge approach with a specific type. By the default, the strategic
+      merge will typically be used.
+    - For example, Custom Resource Definitions typically aren't updatable by the usual strategic merge. You may
+      want to use C(merge) if you see "strategic merge patch format is not supported"
+    - See U(https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/#use-a-json-merge-patch-to-update-a-deployment)
+    - Requires openshift >= 0.6.2
```

Comment:
I think we need to make it more strict since KubeVirt was not there. Let's do it in separate PR.